app/api/controllers/requestParsers/validators/Validator.scala

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
```

Comment:
I've moved the v2 validations into api (ensuring that they are not version specific). i have not replaced the use of v1 validations with these are there are some inconsistencies between v1 and validation that aren't worth spending the time to resolve since v1 is deprecated